pages_domains.go

```diff
@@ -42,8 +42,11 @@ type PagesDomain struct {
 	VerificationCode string     `json:"verification_code"`
 	EnabledUntil     *time.Time `json:"enabled_until"`
 	Certificate      struct {
-		Expired    bool       `json:"expired"`
-		Expiration *time.Time `json:"expiration"`
+		Subject         string     `json:"subject"`
+		Expired         bool       `json:"expired"`
+		Expiration      *time.Time `json:"expiration"`
```

Comment:
@PatrickRice-KSC is `Expiration` still a valid field? It's not listed in the docs: https://docs.gitlab.com/ee/api/pages_domains.html#single-pages-domain

Reply:
@timofurrer  - Weirdly, there are some other undocumented fields that still show up when I run test cases against a local GitLab (example: verified/verification code)... let me go ruby digging to see what the valid fields are.

Reply:
@timofurrer  - Looks like it is, and there may be another `enabled_until` field here as well. Given that's undocumented, I don't think we should _add_ it in the API here, but I think we should leave expiration alone even though it's undocumented:

- https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/api/entities/pages_domain_basic.rb
- https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/api/entities/pages_domain_certificate_expiration.rb

Reply:
In general we like to stick to whatever is in the docs and not add undocumented fields. But as this field is already in I guess this is good to go 👍🏻